src/metabase/analytics/prometheus.clj

```diff
@@ -120,36 +120,22 @@
    :numBusyConnections {:label       "c3p0_num_busy_connections"
                         :description (deferred-trs "C3P0 Number of busy connections")}})
 
-(defn- ->array
-  "Return an array."
-  ^ArrayList
-  [^java.util.Collection coll]
-  (ArrayList. coll))
-
 (defn- stats->prometheus
   "Create an ArrayList of GaugeMetricFamily objects containing measurements from the c3p0 stats. Stats are grouped by
   the property and the database information is attached as a label to multiple measurements of `:numConnections`."
   [stats]
-  (transduce
-   (comp (filter (fn [[raw-label _measurements]]
-                   (or (label-translation raw-label)
-                       (log/warn (trs "Unrecognized measurement {0} in prometheus stats"
-                                      raw-label)))))
```

Comment:
This warning got lost in the translation. I'm not sure how important it is.

Reply:
good catch. It's not super important but would be nice to have the feedback that we don't track that measurement.